FloconDesktop/domain/src/commonMain/kotlin/io/github/openflocon/domain/network/models/FloconNetworkCallDomainModel.kt

```diff
@@ -55,14 +56,21 @@ sealed interface FloconNetworkCallDomainModel {
 }
 
 fun FloconNetworkCallDomainModel.httpCode(): Int? {
-    return when(this) {
+    return when (this) {
         is FloconNetworkCallDomainModel.Http -> this.response?.httpCode
         is FloconNetworkCallDomainModel.GraphQl -> this.response?.httpCode
         else -> null
     }
 }
 
+data class FloconNetworkCallIdDomainModel(
+    val floconCallId: String? = null,
+)
+
 data class FloconNetworkRequestDomainModel(
+    val floconCallId: String? = null,
```

Comment:
as it's a domain model

you should not have a nullable `floconCallId` and `floconNetworkType` 

if the data model does not contains this, returns null in the mapper